chrome_manifest_v3/class_chrome.js

```
@@ -0,0 +1,144 @@
+      errorPrefix: 'Chrome API Error:',
+    };
+    return ck;
+  }
+
+  get ck() {
+    return Chrome.ck;
+  }
+
+  constructor() {
+    // No work needed here currently
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)